### PR TITLE
Digisam 99 solr requests

### PR DESCRIPTION
**Pull request by jara2**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,10 @@
     "core-js": "^2.6.5",
     "vue": "^2.6.10",
     "vue-awesome": "^3.5.4",
+    "vue-i18n": "^8.14.1",
+    "vue-json-pretty": "^1.6.2",
```
Comment (jorntx) [inline package.json]: I hope this is just temporary...:)

File: src/components/Results.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <!-- Example row of columns -->
+    <div class="row">
+      <div class="col-md-4" />
+      <div class="col-md-8">
+        <vue-json-pretty
+          v-if="searchResult"
+          :path="'res'"
+          :data="searchResult"
+          :highlightMouseoverNode="true" />
+      </div>
```
Comment (jorntx) [inline src/components/Results.vue]: You took the time to style JSON - going that extra mile to nice up the demo :-)

File: src/store/search-store.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const state = {
 }
 
 const actions = {
-  doSearch ({ commit }, params) {
+  async doSearch ({ commit }, params) {
```
Comment (jorntx) [inline src/store/search-store.js]: Why du you need the async attribute here?

File: package.json (hunk 1 above)
Comment (jara2) [reply package.json]: yes it is

File: src/store/search-store.js (hunk 3 above)
Comment (jara2) [reply src/store/search-store.js]: Removed it. Seems no need.

**Comment by jara2** [comment]
I merged latest changes from 121 into this also.

**Comment by jara2** [comment]
Merged develop

**Pull request merged**